src/framework/components/element/component.js

```diff
@@ -665,6 +665,8 @@ pc.extend(pc, function () {
                 this.system.app.scene.layers.on("add", this.onLayerAdded, this);
                 this.system.app.scene.layers.on("remove", this.onLayerRemoved, this);
             }
+
+            this.fire("enableelement");
```

Comment:
Not sure I'm super happy about this. I think we may need a more general purpose event for this. But for now I think we should let it pass. As long as it is not a public event.